src/services/email/email-provider.factory.ts

```
@@ -0,0 +1,52 @@
+  static resetInstance(): void {
+    this.instance = null
+  }
+
+  /**
+   * Forces the use of a specific provider (useful for testing)
+   * @param provider - The email provider to use
+   */
+  static setProvider(provider: IEmailProvider): void {
+    this.instance = provider
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)